### ISSUE TITLE
Issues with hardcoded temporary directory

### ISSUE THREAD

**Issue by JensRantil** [rythmengine/rythmengine#354]
We have multiple JVMs running with different users on the same machine. Each JVM uses Rythm and all but one throws an exception on startup because it can't access the Rythm temporary directory, which is hardcoded to use the same directory.

Also, securitywise, isn't it possible for someone to inject a Java code to be executed if they place it in the temporary directory? If so, that should be considered a security issue.

Proposed solution: Create a temporary directory with a random name. Make sure only the process has access to it.

**Comment by greenlaw110** [comment]
You can configure temporary dir for different rythmengine instance (note not for different JVM). Checkout http://rythmengine.org/doc/configuration.md#home_tmp_dir

**Comment by JensRantil** [comment]
@greenlaw110 Thanks. Yes, that's what I did and it worked around the issue. Do you have any input on my security comment and proposed solution?

**Comment by greenlaw110** [comment]
@JensRantil yes I agree with you. Creating a random temp dir should be an nice and neat way. However I am crazy busy at http://actframework.org at the moment, aim to get it released by Mar. I am afraid there is no room for Rythm in at least 2 or 3 months
